brainiak/fcma/cython_blas.pyxbld

```diff
@@ -0,0 +1,13 @@
+def make_ext(modname, pyxfilename):
+    from distutils.extension import Extension
+    from distutils import sysconfig
+    import os
+    opts = []
+    if ('CC' in os.environ and 'icc' in os.environ['CC']) or \
+        'icc' in sysconfig.get_config_var('CC'):
+            opts += ['-lirc', '-lintlc']
```

Comment:
They are to fix an Intel compiler related library link issue
https://github.com/sneumann/mzR/issues/28#issuecomment-149526064